File: 201116-Leet1-TwoSum/seokwon.py
class Solution:
    def twoSum(self, nums, target):
        idxList = []
        for idx, num in enumerate(nums):
            idxList.append((num,idx))
        i = 0
        j = len(nums)-1
        idxList.sort(key=lambda x: x[0])
        print(idxList)
        while i<j:
            result = idxList[i][0]+idxList[j][0]
            if result < target:
                i +=1
            elif result > target:
                j -=1
            else:
                return [idxList[i][1],idxList[j][1]]

File: 201116-Leet1-TwoSum/Seokwon.java
class Solution {
    int SIZE = 2;
    public int[] twoSum(int[] nums, int target) {
        int sum, i = 0;
        int[] reValue = new int[SIZE];
        while(i != nums.length-1){
            for(int j=i+1;j<nums.length;j++){
                sum = nums[i] + nums[j];
                if(sum == target){
                    reValue[0] = i;
                    reValue[1] = j;   
                }
            }
            i++;
        }
           return reValue;
    }
}
